src/Gateway/Blocks/PayplugOney4xWithoutFees.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Payplug\PayplugWoocommerce\Gateway\Blocks;
+
+use Payplug\PayplugWoocommerce\PayplugWoocommerceHelper;
+
+class PayplugOney4xWithoutFees extends PayplugOney3xWithoutFees {
+
+	/**
+	 * Payment method name/id/slug.
+	 *
+	 * @var string
+	 */
+	protected $name = "oney_x4_without_fees";
+
+	protected $icon = 'x4_without_fees_';
+
+	public function oney_enabled() {
```

Comment:
this should be name `get_payment_method_data` shouldn't it?

Reply:
it is being called on the as `return $this->oney_enabled()` on the `get_payment_method_data` in the parent class